src/main/resources/default/assets/stylesheets/biz.scss

```diff
@@ -116,6 +116,21 @@ input.form-control.mls-input {
     overflow: auto;
   }
 
+  .dropdown-menu > li > a:hover,
```

Comment:
sollen wir das ganze mls modal in eine eigene scss auslagern?

Reply:
Klingt sinnvoll